### PR TITLE
Feature/find the largest number in a sequence of n numbers

### PR DESCRIPTION
**Pull request by nammon72**
1. What did you do? (mandatory)
- create 2 files to find the largest number
2. Why would you do that? (mandatory)
- one file to find the largest number among three numbers by "for" loop and if-else to check the condition
- one file to unit test for function
3. An explanation about your `solution or decision or choise` (if needed)

4. Checklist (if needed)

5. Action need to do before merge (if needed)

6. Action need to do after merge (if needed)

7. Others note (if needed)

### CODE REVIEW

File: src/largest-number-in-sequence.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// const numberArray = [-1, 4, 2, 4, 6, 3, 1, 6, 7, 5, 300, 22, 100];
+function findLargestNumber(numberArray) {
+  let max = 0;
+  for (let index = 0; index < numberArray.length; index++) {
+    if (max < numberArray[index]) {
+      max = numberArray[index];
+    } else {
+      max = max;
```
Comment (anhdiepmmk) [inline src/largest-number-in-sequence.js:8]: why need that? => remove this

File: src/largest-number-in-sequence.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const largestNumber = require('./largest-number-in-sequence');
```
Comment (anhdiepmmk) [inline src/largest-number-in-sequence.test.js:1]: add a test for case numberArray is empty []

File: src/largest-number-in-sequence.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// const numberArray = [-1, 4, 2, 4, 6, 3, 1, 6, 7, 5, 300, 22, 100];
```
Comment (anhdiepmmk) [inline src/largest-number-in-sequence.js:1]: remove this

File: src/largest-number-in-sequence.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+function findLargestNumber(numberArray) {
```
Comment (anhdiepmmk) [inline src/largest-number-in-sequence.js:2]: `findMax` but `findLargestNumber` will be fine. also numberArray can be named `numbers`

**Review by anhdiepmmk — CHANGES_REQUESTED** [review]

File: src/display-fibonacci-sequence.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+function displayFibonacci(number) {
```
Comment (anhdiepmmk) [inline src/display-fibonacci-sequence.js:1]: we are working on `find the largest number in a sequence of n numbers` task so my question is why we need fibonacci in this ticket ? 
So pls remove this due to unrelated code

**Review by anhdiepmmk — CHANGES_REQUESTED** [review]

**Comment by anhdiepmmk** [comment]
@nammon72 pls resolve conflicts and fix all lint error
